### DISCUSSION TITLE
Network connection to the Guacamole server appears unstable

### DISCUSSION THREAD

**Discussion by fuchen61** [neurodesk/neurodesk.github.io-hugo#445] [Q&A] [answered]
There is "Network connection to the Guacamole server appears unstable", and the Neurodesk app is interrupted for multiple times, and no module can be opened.
![1](https://github.com/user-attachments/assets/7831aab8-4300-4818-b532-72a791347304)
![2](https://github.com/user-attachments/assets/0dc1cbbc-0039-4ed4-9cca-7961ee87b748)

**Comment by stebo85** [comment]
That's caused from a slow network connection. Where are you seeing this? Is this on our play servers? Or a version you run locally?

**Reply by fuchen61** [reply]
> That's caused from a slow network connection. Where are you seeing this? Is this on our play servers? Or a version you run locally?

Hello, I saw this on the newest version on my local computer for several times recently. However, the network of my local computer is normal.

**Reply by stebo85** [reply]
Ok, then it could be something related to your docker network. Have you tried restarting docker or your computer?

**Reply by fuchen61** [reply]
> Ok, then it could be something related to your docker network. Have you tried restarting docker or your computer?

Yes, it caused all the modules can not be opened in Neurodesk GUI. I have reinstall the docker, but the issue remains.

**Reply by stebo85** [reply]
Ok, then this could be a recent docker bug that causes network issues. A few things you can try:
1) start it with a manual dns server set: add this to your manual Neurodesk docker start command:   --dns 8.8.8.8 \
2) update docker to the latest version
3) uninstall and reinstall docker 

Which operating system is this on? Which docker version do you have?

**Reply by fuchen61** [reply]
> Ok, then this could be a recent docker bug that causes network issues. A few things you can try:
> 
> 1. start it with a manual dns server set: add this to your manual Neurodesk docker start command:   --dns 8.8.8.8 \
> 2. update docker to the latest version
> 3. uninstall and reinstall docker
> 
> Which operating system is this on? Which docker version do you have?

I'm using the Ubuntu 20.04. And the Neurodesk version is the latest one.

**Reply by stebo85** [reply]
Ok, then another great option for you to try is: instead of docker, use podman.

**Reply by fuchen61** [reply, marked as answer]
> Ok, then another great option for you to try is: instead of docker, use podman.

Hi, I restarted the computer, and the issue is now resolved (still using docker). Thank you for your suggestion!
